app/src/components/inputs/inputRadio/inputRadio.jsx

```diff
@@ -31,27 +31,31 @@ export const InputRadio = ({
   onFocus,
   onBlur,
   mobileDisabled,
-}) => (
-  // eslint-disable-next-line
-  <label
-    className={cx('input-radio', { disabled, 'mobile-disabled': mobileDisabled })}
-    onFocus={onFocus}
-    onBlur={onBlur}
-    tabIndex="1"
-  >
-    <input
-      type="radio"
-      className={cx('input')}
-      disabled={disabled}
-      onChange={onChange}
-      value={ownValue}
-      checked={value === ownValue}
-      name={name}
-    />
-    <span className={cx('toggler', { checked: value === ownValue, 'at-top': circleAtTop })} />
-    {children && <span className={cx('children-container')}>{children}</span>}
-  </label>
-);
+  checked,
+}) => {
+  const isChecked = checked || value === ownValue;
```

Comment:
I suggest don't overload component with additional `checked` props. `value` and `ownValue` are enough.